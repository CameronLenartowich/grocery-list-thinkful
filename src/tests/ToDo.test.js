import React from 'react';
import ReactDOM from 'react-dom';
import ToDo from '../components/Todo';
import { exportAllDeclaration } from '@babel/types';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
Enzyme.configure({ adapter: new Adapter() });


it('render correctly todo component', () => {  
    const component = renderer.create(<ToDo />).toJSON();
    expect(component).toMatchSnapshot();
});