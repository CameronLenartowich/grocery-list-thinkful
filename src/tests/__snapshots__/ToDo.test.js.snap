// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly text component 1`] = `
<li>
  <input
    type="checkbox"
  />
  <span />
</li>
`;

exports[`render correctly todo component 1`] = `
<li>
  <input
    type="checkbox"
  />
  <span />
</li>
`;
